.github/workflows/other.yml

```
@@ -76,4 +76,4 @@ jobs:
           docker buildx build \
             --platform linux/amd64 \
             --push \
-            -t ${{ secrets.DOCKER_HUB_USERNAME }}/techeerism-spring:$DOCKER_IMAGE_TAG .
+            -t ${{ secrets.DOCKER_HUB_USERNAME }}/techeerism-spring:$DOCKER_IMAGE_TAG .
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)